index.html

```diff
@@ -7,5 +7,7 @@
     <link rel="stylesheet" href="style.css" />
     <title>Hello Microverse</title>
   </head>
-  <body></body>
+  <body>
+    <h1>Hello Microverse</h1>
```

Comment:
- [ ] As stated the project requirement Kindly make sure that A .css file that includes `one class` with styles for your header (make it in your favorite color).
- ![Capture](https://user-images.githubusercontent.com/93598189/211501647-d22fcab1-eabd-46f6-84a6-f12672abe6f9.PNG)